package.json

```diff
@@ -2,10 +2,26 @@
   "name": "root",
   "private": true,
   "scripts": {
-    "start": "lerna run start --parallel"
```

Comment:
Can we add other commands to build test checks and lint. Al, these are missing.

Reply:
okay, I thought test is not required, I will add it.